cmd/tether/ops_linux_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
I wonder if implementing all this logic makes sense. We could use go-mock to generate code that could pre-precord expected behavior, so you can easily test error cases, etc.